src/components/Courses.jsx

```diff
@@ -19,6 +32,17 @@ const Courses = () => {
         });
   }, [URL]);
 
+  const handleClick = async (e, code) => {
+    e.preventDefault();
```

Comment:
not sure this is needed. Is there a default behaviour of this button you are trying to prevent?